### DISCUSSION TITLE
Różnica w wartościach atrybutów z sensorów "balance" oraz sensorów "consumption" - "generation"

### DISCUSSION THREAD

**Discussion by sch7dom** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#157] [General]
Proszę o wyjaśnienie skąd są różnica między wartością atrybutów (w nawiasach podałem mój aktualny stan)

atrybut "sum_generation" ( -74.747)   z sensora "monthly_balance" a wartością sensora "monthly_energy_generation" (91.891)

oraz

atrybut "sum_consumption" (113.607) z sensora  "monthly_balance" a wartością sensora "monthly_energy_consumption" (130.751)

przy czym sumy się zgadzają "sum_generation" (-74.747) + "sum_consumption"  (113.607) = "monthly_energy_consumption" (130.751) -  "monthly_energy_generation" (91.891)   =  wartość sensora "balance" (38.86)

Dziwnie to wygląda, że jak patrzę np. na atrybut  "sum_consumption"  sensora "monthly_balance" to mam wartość 113,607, a na sensorze "monthly_energy_consumption" to jest wartość 94,891  pomijając znaki  to jest ok 20 różnicy...  która wartość zużycia jest prawidłowa, skąd ta różnica? 

Dziękuje 
Marcin

**Comment by PiotrMachowski** [comment]
Sensory "balance" zawierają dane po przeliczeniu balansowania godzinowego. Więcej info tu:
https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/issues/57
